### ISSUE TITLE
SensorID changed

### ISSUE THREAD

**Issue by TCThenna** [eudaq/eudaq#350]
Dear all,

the telescope data taken last year included the telescope sensor IDs  from 0 to 5:
--------------- print out of TrackerData collection --------------- 

  flag:  0x0
     LCIO::TRAWBIT_ID1    : 0
 parameter CellIDEncoding [string]: sensorID:7,sparsePixelType:5, 

 [   id   ] |  cellid0 |  cellid1 |   time   | cellid-fields  
------------|----------|----------|----------|----------------
 [00000012] | 00000256 | 00000000 |        0 | sensorID:0,sparsePixelType:2chargeADC : 1140,98,1,0,897,232,1,0,1062,523,1,0,
 [00000013] | 00000257 | 00000000 |        0 | sensorID:1,sparsePixelType:2chargeADC : 54,19,1,0,1143,83,1,0,1144,83,1,0,1143,84,1,0,1144,84,1,0,1143,85,1,0,1144,85,1,0,567,241,1,0,123,378,1,0,
 [00000014] | 00000258 | 00000000 |        0 | sensorID:2,sparsePixelType:2chargeADC : 485,10,1,0,1143,87,1,0,1144,87,1,0,1143,88,1,0,1144,88,1,0,159,434,1,0,
 [00000015] | 00000259 | 00000000 |        0 | sensorID:3,sparsePixelType:2chargeADC : 438,91,1,0,1105,91,1,0,1105,92,1,0,1103,370,1,0,1103,371,1,0,222,399,1,0,
 [00000016] | 00000260 | 00000000 |        0 | sensorID:4,sparsePixelType:2chargeADC : 283,51,1,0,1099,88,1,0,1100,88,1,0,1100,89,1,0,275,216,1,0,291,391,1,0,197,495,1,0,1069,497,1,0,
 [00000017] | 00000261 | 00000000 |        0 | sensorID:5,sparsePixelType:2chargeADC : 578,64,1,0,1098,78,1,0,1099,78,1,0,1098,79,1,0,1099,79,1,0,350,111,1,0,1027,214,1,0,182,273,1,0,395,393,1,0,396,393,1,0,403,393,1,0,190,428,1,0,
------------|----------|----------|----------|----------------

Whereas this year, the sensorIDs are numbered 1 to 6:

--------------- print out of TrackerData collection --------------- 

  flag:  0x0
     LCIO::TRAWBIT_ID1    : 0
 parameter CellIDEncoding [string]: sensorID:7,sparsePixelType:5, 

 [   id   ] |  cellid0 |  cellid1 |   time   | cellid-fields  
------------|----------|----------|----------|----------------
 [00000012] | 00000257 | 00000000 |        0 | sensorID:1,sparsePixelType:2chargeADC : 855,409,1,0,856,409,1,0,257,433,1,0,
 [00000013] | 00000258 | 00000000 |        0 | sensorID:2,sparsePixelType:2chargeADC : 897,431,1,0,897,432,1,0,
 [00000014] | 00000259 | 00000000 |        0 | sensorID:3,sparsePixelType:2chargeADC : 1065,22,1,0,967,277,1,0,159,434,1,0,936,474,1,0,
 [00000015] | 00000260 | 00000000 |        0 | sensorID:4,sparsePixelType:2chargeADC : 898,20,1,0,899,20,1,0,1072,63,1,0,1071,64,1,0,1072,64,1,0,993,507,1,0,
 [00000016] | 00000261 | 00000000 |        0 | sensorID:5,sparsePixelType:2chargeADC : 889,70,1,0,890,70,1,0,389,356,1,0,1012,544,1,0,1012,545,1,0,1013,545,1,0,
 [00000017] | 00000262 | 00000000 |        0 | sensorID:6,sparsePixelType:2chargeADC : 884,117,1,0,884,118,1,0,885,118,1,0,831,268,1,0,395,393,1,0,403,393,1,0,

This is suboptimal, as many eutel processors check if the first ID is == 0.

I am trying to find out, where this happened, but couldnt find the root of this. 

Any ideas?

Cheers,
Hendrik

**Comment by tbisanz** [comment]
It should be set by the IDs set in the BORE:

unsigned id = from_string(bore.GetTag("ID" + to_string(i)), i);

**Comment by TCThenna** [comment]
last year, we even had strings as the parameter value
https://github.com/eudaq/eudaq-configuration/blob/master/duranta/configuration/telescope_beam.conf

but it dont recall this to be a problem. Why did it even work?

in the  NIConverterPlugin.cc, we do (L114f)
unsigned id = board;
if (id < m_ids.size())
  id = m_ids[id];
and board is just a counter here starting at 0.
This id is used to create the standardplane.

Now, Jan changed all files to read 1 ... 6.
https://github.com/eudaq/eudaq-configuration/commit/9dd089b5a6fbfaf4d76c7128ff5f3ff13af6efb2

But standardplanes do not do anything to the given (like forcing it to 0 ..5). So I am still puzzled why this has worked before ... 

Bottom line, we might have to change the config files, and users will have severe problems when analysing this years data ...
